docs/design-proposals/node/Unified-Vertical-Scaling-for-Container-and-VM-Pods.md

```diff
@@ -319,3 +341,129 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Changes in Arktos-vm-runtime-service (CRI for VM workload types)
+
+#### Specific for VM workload
+
+1. Hotplug(hotUnplug) vs Memory Ballooning driver for Memory resizing
+
+   Currently per our knowledge these are two technologies that supports live memory resizing for VMs. Both have been there for
+   Linux for years, and both has pros and cons in supporting memory resizing for VMs. There are also some proposals to combine
+   them to better support VM resizing for Linux VMs.
+
```

Comment:
Can you elaborate this part? Curious to know the key reason why we choose memory hotplug over balooning.

Reply:
updated the doc.